packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/flush-archive/flush-archive.controller.js

```diff
@@ -0,0 +1,77 @@
+import {
+  COLD_ARCHIVE_DEFAULT_REGION,
+  TERMINATE_INPUT_PATTERN,
+} from './flush-archive.constants';
+
+import { COLD_ARCHIVE_CONTAINER_STATUS } from '../containers.constants';
+
+export default class ColdArchiveContainersFlushArchiveController {
+  /* @ngInject */
+  constructor($translate, PciStoragesColdArchiveService) {
+    this.$translate = $translate;
+    this.pciStoragesColdArchiveService = PciStoragesColdArchiveService;
+  }
+
+  $onInit() {
+    this.trackPage('containers::container::flush-archive');
+
+    this.isLoading = false;
+    this.TERMINATE_INPUT_PATTERN = TERMINATE_INPUT_PATTERN;
+    this.COLD_ARCHIVE_CONTAINER_STATUS = COLD_ARCHIVE_CONTAINER_STATUS;
+  }
+
+  getHeadingContent() {
+    return this.$translate.instant(
+      'pci_projects_project_storages_containers_container_cold_archive_flush_title',
+      {
+        containerName: this.container.name,
+      },
+    );
+  }
```

Comment:
**nitpick:** Directly use `data-translate` in the template.

Reply:
Done.